src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -2537,6 +2537,22 @@ public boolean locatorsUpdateCopy() throws SQLException {
         checkClosed();
         return true;
     }
+
+    /* -------------- MSSQL-JDBC Extension methods  ---------------- */
+
```

Comment:
please add javadoc header (see other public api's for format)

Reply:
Done.